Celeste.Mod.mm/Patches/EntityData.cs

```diff
@@ -12,5 +12,11 @@ class patch_EntityData : EntityData {
             return orig_Has(key);
         }
 
+        public EntityID EntityID;
+
+        internal void SetEntityID(bool? isTrigger = null) {
```

Comment:
I find the name weird since it is not a setter for EntityID 🤔 maybe `ComputeEntityID` or `InitializeEntityID`?

Reply:
Is the parameter ever used?

Reply:
the parameter is not used... yet. it's added in for a future feature, assuming we infact get that far with this PR lmao
I'll change the name to InitializeEntityID